rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionCommand.java

```diff
@@ -12,7 +12,14 @@
 
     public SolutionCommand( CommandAction... actions )
     {
-        this.actions = actions;
+        if ( 0 == actions.length )
+        {
+            this.actions = new CommandAction[] { new WaitAction( 1 ) };
+        }
+        else
+        {
+            this.actions = actions;
+        }
     }
```

Comment:
if time steps but nothing has been done the default is a wait of 1. like just hitting enter and typing nothing in the text UI. Made sense to me.